.github/workflows/tests.yml

```diff
@@ -42,14 +65,13 @@ jobs:
       - name: Add localsettings
         run: cp evap/settings_test.py evap/localsettings.py
       - name: Run tests
-        run: ${{ matrix.command }}
-
+        run: python manage.py test --shuffle
 
   mypy:
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-20.04
 
     container:
-      image: python:3.7
+      image: python:3.8
```

Comment:
Indeed, apache gives the "It works!" apache default page, because reloading the config fails. I wonder why in the year 2022 it's still necessary to point apache so some shared library it should load, but the [current version in the mod_wsgi documentation](https://modwsgi.readthedocs.io/en/develop/user-guides/quick-installation-guide.html) seems to be even worse, asking us to compile the module from source (and then loading it similarly to what we do in wsgi.template.conf).

Reply:
turns out there is a [`libapache2-mod-wsgi-py3`](https://packages.ubuntu.com/focal/libapache2-mod-wsgi-py3) package that can do all this for us -- but its not linked in any documentation (mod-wsgi, django, interwebz tutorials, ...) ugh.